pkg/stomp/StompProducer.php

```diff
@@ -39,7 +40,18 @@ public function send(Destination $destination, Message $message): void
 
         $stompMessage = new StompLibMessage($message->getBody(), $headers);
 
-        $this->stomp->send($destination->getQueueName(), $stompMessage);
+        try {
+            $this->stomp->send($destination->getQueueName(), $stompMessage);
+        } catch (ConnectionException $ex) {
+            if (!$this->stomp->isConnected()) {
```

Comment:
what this `if` for?

Reply:
I had originally placed it there so that if something explicitly closed the connection, it would be respected (rather than accidentally forcing it back on).  Basically to give _some_ hope for integrity of the state when it's `false`.

(It's just `true` that can't be trusted.)